src/analysis/plugin/plugin.py

```diff
@@ -106,34 +110,34 @@ def summarize(self, result: Schema) -> list[str]:
     def analyze(
         self,
         file_handle: io.FileIO,
-        virtual_file_path: dict,
+        virtual_file_path: VfpDict,
         analyses: dict[str, pydantic.BaseModel],
-    ) -> typing.Optional[Schema]:
+    ) -> Schema | None:
         """Analyze a file.
         May return None if nothing was found.
 
         :param file_handle: :py:class:`io.FileIO` instance of the file to be analyzed
         :param virtual_file_path: The virtual file paths, see :py:class:`~objects.file.FileObject`
         :param analyses: A dictionary of dependent analysis
 
-        :return: The analysis if anything was found.
+        :return: The analysis results (if there are any).
```

Comment:
nit: why use parentheses?

Reply:
In general? :sweat_smile:  

I don't know, if you want I can remove them. I originally just wanted to specify that analysis *results* are returned.

Reply:
I meant that what is written in the parentheses is useful information and reads better without the parantheses.